css/menu.css

```diff
@@ -26,7 +26,6 @@ body.has-menu-left.has-push-menu {
 html.no-android body.has-push-menu,
 html.no-android.has-push-menu {
   -webkit-overflow-scrolling: auto;
-  overflow: visible;
```

Comment:
I think the entire block for `html.no-android body.has-push-menu, html.no-android.has-push-menu` can probably be deleted because if `overflow: visible` isn't applied, `-webkit-overflow-scrolling: auto;` probably serves no purpose too. Does it make a difference in your testing? Is there a reason why you're keeping it?